docs/UserGuide.md

```diff
@@ -112,62 +140,158 @@ the error message shown would be based on the leftmost invalid prefix parameter
 
 ## General Layout
 
-The following is a description of the general layout of nustracker.
+The following is a description of the general layout of **nustracker**.
 
-![general-layout](images/general-layout.png)
+![general-layout](images/user-guide/general-layout.png)
 
 1. **Top bar**
-    
-    The top bar consists of the buttons: **Refresh, Toggle theme, Settings, Students, Events, Help, and Exit**.
-    * **Refresh**
-      
+ 
+    * **Refresh**<br>
         Refreshes the currently displayed students/events. Press this button when you add profile pictures.
 
-    * **Toggle Theme**
-        
-        Toggles the theme of nustracker, from Dark theme to Light theme, and vice versa.
-
-    * **Settings**
+    * **Toggle Theme**<br>
+        Toggles the theme of **nustracker**, from Dark theme to Light theme, and vice versa.
 
+    * **Settings**<br>
          Opens the Settings Window.
      
-    * **Students**
-
+    * **Students**<br>
         Displays current students.
 
-   * **Events**
+    * **Events**<br>
+        Displays current events.
+
+    * **Help**<br>
+        Opens the Help window.
+
+    * **Exit**<br>
+      Exit **nustracker**.
+
+3. **Command bar**
+ 
+    This space is for the user (you) to type in the commands. For more information about commands, see the section below.
```

Comment:
Maybe you can add a link to "see the section below" that links to the commands section